### PR TITLE
Continuation of "semi support for touchscreens + clickRegion"

### PR DESCRIPTION
**Pull request by thrailkillr**
I saw the other PR and that it was stale, so I took it upon myself to make the cosmetic changes you requested.

Closes #5

### CODE REVIEW

File: button.lua [hunk 1]
```diff
@@ -20,31 +22,36 @@ function PrimeUI.button(win, x, y, text, action, fgColor, bgColor, clickedColor)
...
     bgColor = expect(7, bgColor, "number", "nil") or colors.gray
     clickedColor = expect(8, clickedColor, "number", "nil") or colors.lightGray
+    periphName = expect(9, periphName, "string", "nil")
     -- Draw the initial button.
     win.setCursorPos(x, y)
     win.setBackgroundColor(bgColor)
     win.setTextColor(fgColor)
     win.write(" " .. text .. " ")
     -- Get the screen position and add a click handler.
+    local screenX, screenY = PrimeUI.getWindowPos(win, x, y)
```
Comment (MCJack123) [inline button.lua]: Could you move this back down into the task function where it was before? Not sure why this migrated back up here.

File: clickRegion.lua [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+function PrimeUI.clickRegion(win, x, y, width, height, action, periphName)
+    expect(1, win, "table")
+    expect(2, x, "number")
+    expect(3, y, "number")
+    expect(4, width, "number")
+    expect(5, height, "number")
+    expect(6, action, "function", "string")
+    periphName = expect(7, periphName, "string", "nil")
+    -- Get the screen position and add a click handler.
+    local screenX, screenY = PrimeUI.getWindowPos(win, x, y)
```
Comment (MCJack123) [inline clickRegion.lua]: Same as above.

File: clickRegion.lua [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    PrimeUI.addTask(function()
+        local buttonDown = false
+        while true do
+            local event, button, clickX, clickY = os.pullEvent()
+            if (event == "monitor_touch" and periphName == button)
+                or (event == "mouse_down" and button == 1 and periphName == nil) then
+                -- Finish a click event.
+                if clickX >= screenX and clickX < screenX + width
+                    and clickY >= screenY and clickY < screenY + height + 1 then
```
Comment (MCJack123) [inline clickRegion.lua]:
```suggestion
                    and clickY >= screenY and clickY < screenY + height then
```

**Review by MCJack123 — CHANGES_REQUESTED** [review]
Looks mostly good, just a few small tweaks to resolve a regression in window positions.

**Comment by thrailkillr** [comment]
Changes have been made.

**Pull request merged**